docs/layers/geojson-layer.md

```diff
@@ -168,11 +168,12 @@ Only works if `getLineDashArray` is specified.
 
 Whether the layer should be rendered in high-precision 64-bit mode. Note that since deck.gl v6.1, the default 32-bit projection uses a hybrid mode that matches 64-bit precision with significantly better performance.
 
-##### `lightSettings` (Object, optional) **EXPERIMENTAL**
+##### `material` (Object, optional)
+
+* Default: `PhongMaterial`
```

Comment:
This looks a little strange? `PhongMaterial` is a string, not an `Object`. 

Maybe define it as a default material?

Reply:
`` is used for hightlight style, PhongMaterial is a class, should I use Default: `new PhongMaterial()`?

Reply:
Better to add `new`  in that case?

`new PhongMaterial()`